### DISCUSSION TITLE
How to dismiss the navbar menu on link click and off click?

### DISCUSSION THREAD

**Discussion by tansanDOTeth** [saadeghi/daisyui#2492] [Q&A] [answered]
Here is a video of it dismissing on off click nor link click

https://github.com/saadeghi/daisyui/assets/92072158/448cfb29-adfd-4fef-bdd2-7faa31e91e40

**Comment by boggye** [comment]
I am interested in this as well. If you go to the [navbar](https://daisyui.com/components/navbar/) samples and go to the one that has some buttons with dropdown menus, once displayed the menus can be closed only as long as you click on the chevron button. On the other hand if I look at the main components [page](https://daisyui.com/components/), and expand the Theme dropdown, that list is discarded as soon as I click somewhere else on the screen.

Interestingly enough, in the [dropdown](https://daisyui.com/components/dropdown/) samples, in the majority of the samples the dropdown menu is discarded when click somewhere else on the screen.

I would appreciate if you can provide some guidance. 🙏

Update: I found this sample: https://daisyui.com/components/dropdown/#dropdown-in-navbar which does work as expected. I guess the only thing left is to include the chevron icon. I will keep looking.

Thank you

**Reply by saadeghi** [reply, marked as answer]
It is explained in dropdown page: https://daisyui.com/components/dropdown/

Method 1: using details and summary tags  
Stays open until gets clicked again. Or you can close it using JS by removing the `open` attribute

Method 2: Using CSS focus
Closes when focus is lost

**Reply by tbias25** [reply]
Hello @saadeghi,

could you give me please a example to use the second method?
